lineage/figures/figure4.py

```diff
@@ -1,133 +1,108 @@
-            obsBernoulliG2.append(cell.obs[1])
-            obsG1.append(cell.obs[2])
-            obsG2.append(cell.obs[3])
-
-            if pred_states_by_lineage[indx][cell_ind] == 0:  # if the cell is in state 1
-                obsBernoulliG1S1.append(cell.obs[0])
-                obsBernoulliG2S1.append(cell.obs[1])
-                obsG1S1.append(cell.obs[2])
-                obsG2S1.append(cell.obs[3])
-            else:  # if the cell is in state 2
-                obsBernoulliG1S2.append(cell.obs[0])
-                obsBernoulliG2S2.append(cell.obs[1])
-                obsG1S2.append(cell.obs[2])
-                obsG2S2.append(cell.obs[3])
-
-    list_bern_g1 = [obsBernoulliG1, obsBernoulliG1S1, obsBernoulliG1S2]
-    list_bern_g2 = [obsBernoulliG2, obsBernoulliG2S1, obsBernoulliG2S2]
-
-    totalObsG1 = pd.DataFrame(columns=['G1 phase duration [hr]', 'state'])
-    totalObsG1['G1 phase duration [hr]'] = obsG1 + obsG1S1 + obsG1S2
-    totalObsG1['state'] = ['total'] * len(obsG1) + ['state 1'] * len(obsG1S1) + ['state 2'] * len(obsG1S2)
-    totalObsG2 = pd.DataFrame(columns=['G2 phase duration [hr]', 'state'])
-    totalObsG2['G2 phase duration [hr]'] = obsG2 + obsG2S1 + obsG2S2
-    totalObsG2['state'] = ['total'] * len(obsG2) + ['state 1'] * len(obsG2S1) + ['state 2'] * len(obsG2S2)
-
-    return totalObsG1, totalObsG2, list_bern_g1, list_bern_g2
-
-
-def figureMaker2(ax, totalObsG1, totalObsG2, list_bern_g1, list_bern_g2):
+    Calculates accuracy and parameter estimation
+    over an similar number of cells in a lineage for
+    a uncensored two-state model but differing state distribution.
+    We increase the proportion of cells in a lineage by
+    fixing the Transition matrix to be biased towards state 0.
     """
-    Makes the common 6 panel figures displaying parameter estimation across lineages
-    of various types and sizes.
+
+    # Creating a list of populations to analyze over
+    list_of_Ts = [np.array([[i, 1.0 - i], [i, 1.0 - i]]) for i in np.linspace(0.01, 0.5, 50)]
+    list_of_populations = []
+    list_of_fpi = []
+    list_of_fT = []
+    list_of_fE = []
+    list_cell_num_st1 = []
+    for T in list_of_Ts:
+        population = []
+
+        good2go = False
+        while not good2go:
+            tmp_lineage = LineageTree(pi, T, E2, 2**9-1)
+            good2go = lineage_good_to_analyze(tmp_lineage)
+
+        # how many cells are in state 1
+        cell_num_st1 = []
+        for cell in tmp_lineage.full_lineage:
+            if cell.state == 0:
+                cell_num_st1.append(1.0)
+
+        list_cell_num_st1.append(len(cell_num_st1))
+        population.append(tmp_lineage)
+
+        # Adding populations into a holder for analysing
+        list_of_populations.append(population)
+        list_of_fpi.append(pi)
+        list_of_fT.append(T)
+        list_of_fE.append(E2)
+
+    cell_nums, _, _, _, _, _, accuracy = commonAnalyze(list_of_populations, list_of_fT=list_of_fT)
```

Comment:
Just add `xtype="prop"` and it'll turn `cell_nums` into proportion of cells in state 0, I think. The calculation is done in `Results`. See here: https://github.com/meyer-lab/lineage-growth/blob/169ad6601af9fc8732f03bd7ea6b634caf92389a/lineage/Analyze.py#L169 

`commonAnalyze` will return this for your populations, if you use that `xtype`. This works the same way as using`xtype="wass"`.